airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -469,7 +489,7 @@ def get_task_instance_try_details(
     task_id: str,
     task_try_number: int,
     session: Annotated[Session, Depends(get_session)],
-    map_index: int = -1,
+    map_index: Annotated[int, Depends(lambda: -1)],
```

Comment:
Why this change ? This seems odd, what are we trying to achieve ?

Reply:
changed the default value back to `-1`